tracer/src/Datadog.Tracer.Native/dd_profiler_constants.h

```diff
@@ -59,6 +59,13 @@ const shared::WSTRING include_assemblies[]{
     WStr("System.Diagnostics.Process"),
 };
 
+// Note that this list should be kept in sync with the values in shared/src/Datadog.Trace.ClrProfiler.Native/util.h
+const shared::WSTRING default_exclude_assemblies[]{
+    WStr("dd-trace"),
+    WStr("dd-trace.exe"),
+    WStr("aspnet_state.exe"),
```

Comment:
Should we add `sqlservr.exe`? I don't think we ever want to instrument SQL Server.

Reply:
I thought about it, but then I thought... is anyone using us to instrument SQL Server?! 😅

Reply:
I really hope not...

Reply:
Added